src/components/common/Select.js

```diff
@@ -0,0 +1,50 @@
+import React, { PureComponent } from 'react';
+import { string, object } from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { parseInputErrors } from 'utils/helpers';
+
+export default class Select extends PureComponent {
+  static propTypes = {
+    input: object.isRequired,
+    label: string,
+    type: string.isRequired,
+    placeholder: string,
+    meta: object,
+  };
+
+  OptionItem({ value, label }) {
```

Comment:
As a convention only the components names start with a capital letter.

This function should be: `optionItem`